src/views/Entities/IngressDataplanes.vue

```diff
@@ -541,6 +542,8 @@ export default {
               lastUpdated: lastUpdated,
               totalUpdates: totalUpdates,
               totalRejectedUpdates: totalRejectedUpdates,
+              publicAddress: (response.dataplane.networking.ingress.publicAddress || null),
+              publicPort: (response.dataplane.networking.ingress.publicPort || null),
```

Comment:
I don't understand the language here, but is there a case where ingress can be `null` and we get dereference issues?

Reply:
Ingress won't be null, because we are asking API for only ingress data planes (which are defined by having an ingress part).
The difference between
```
response.dataplane.networking.ingress.publicAddress
```
and
```
response.dataplane.networking.ingress.publicAddress || null
```
is that the first one will return `undefined` and the second one `null`. I think Vue when rendering is fine with both, but I see that we nullify all the values

Reply:
Vue will handle the `undefined` but I like to nullify the items out of paranoia and consistency. It's probably overkill but it ensures I know exactly what the fallback will be.